ui/panels-plugin/src/components/stat-chart/StatChart.tsx

```diff
@@ -194,27 +198,38 @@ export function StatChart(props: StatChartProps) {
       series: statSeries,
       textStyle: {
         color: backgroundColor === 'transparent' ? '#000000' : '#FFFFFF',
-        fontSize: 18,
+        fontSize: 25,
         lineHeight: 18,
         fontFamily: '"Lato", sans-serif',
+        fontWeight: 'bold',
       },
       media: [
         {
           query: {
-            maxWidth: 180,
+            maxWidth: 150,
           },
           option: {
             textStyle: {
-              fontSize: 10,
-              lineHeight: 10,
+              fontSize: 12,
+            },
+          },
+        },
+        {
+          query: {
+            minWidth: 150,
+          },
+          option: {
+            textStyle: {
+              fontSize: `max(14px, ${baseFontSize}px)`,
```

Comment:
This is really cool, I actually didn't realize you could use CSS functions in textStyle.fontSize this way